src/components/pages/order/Main/Menu.jsx

```diff
@@ -33,4 +48,28 @@ const MenuStyled = styled.div`
   justify-items: center;
   box-shadow: 0px 8px 20px 8px rgba(0, 0, 0, 0.2) inset;
   overflow: scroll;
+
+  .empty-menu-container {
+    display: flex;
+    width: 1400px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    margin-bottom: 200px;
+    .empty-menu {
+      margin-bottom: 30px;
+      font-size: ${theme.fonts.size.P4};
+      font-family: "Amatic SC";
+      color: ${theme.colors.greySemiDark};
+      font-weight: ${theme.fonts.weights.regular};
+      .title {
+        font-weight: ${theme.fonts.weights.bold};
+      }
+    }
+  }
+  .button {
```

Comment:
```suggestion
  
  .button {
```

saut de ligne pour aérer ton CSS.